libs/akita/src/__tests__/queryEntity.spec.ts

```diff
@@ -383,48 +383,48 @@ describe('Entities Query', () => {
           expect(spy).toHaveBeenCalledTimes(1);
           expect(spy).toHaveBeenCalledWith(false);
           let factory = ct();
-          store.set(factory());
+          // store.set(factory());
+          // expect(spy).toHaveBeenCalledTimes(2);
+          // expect(spy).toHaveBeenCalledWith(true);
+          // store.remove();
+          // expect(spy).toHaveBeenCalledTimes(3);
+          // expect(spy).toHaveBeenCalledWith(false);
+          // store.set(factory());
+          // expect(spy).toHaveBeenCalledTimes(4);
+          // expect(spy).toHaveBeenCalledWith(true);
+          store.setHasCache(true);
           expect(spy).toHaveBeenCalledTimes(2);
           expect(spy).toHaveBeenCalledWith(true);
-          store.remove();
-          expect(spy).toHaveBeenCalledTimes(3);
-          expect(spy).toHaveBeenCalledWith(false);
-          store.set(factory());
-          expect(spy).toHaveBeenCalledTimes(4);
-          expect(spy).toHaveBeenCalledWith(true);
+          // store.set(factory());
+          // expect(spy).toHaveBeenCalledTimes(6);
+          // expect(spy).toHaveBeenCalledWith(true);
+          // store.remove();
+          // expect(spy).toHaveBeenCalledTimes(7);
+          // expect(spy).toHaveBeenCalledWith(false);
           store.setHasCache(false);
-          expect(spy).toHaveBeenCalledTimes(5);
-          expect(spy).toHaveBeenCalledWith(false);
-          store.set(factory());
-          expect(spy).toHaveBeenCalledTimes(6);
-          expect(spy).toHaveBeenCalledWith(true);
-          store.remove();
-          expect(spy).toHaveBeenCalledTimes(7);
+          expect(spy).toHaveBeenCalledTimes(3);
           expect(spy).toHaveBeenCalledWith(false);
           store.setHasCache(true);
-          expect(spy).toHaveBeenCalledTimes(8);
-          expect(spy).toHaveBeenCalledWith(true);
-          store.setHasCache(true);
-          expect(spy).toHaveBeenCalledTimes(8);
+          expect(spy).toHaveBeenCalledTimes(4);
         });
       });
 
```

Comment:
What we're going to do with the commented specs?

Reply:
Some of them are obsolete and should be removed. Others must be adapted to cover the new behaviour. I have commented them out to review them again when all changes have been made.